code/modules/crew_objectives/medical_objectives.dm

```diff
@@ -1,14 +1,13 @@
 /*				MEDICAL OBJECTIVES				*/
 
 /datum/objective/crew/morgue //Ported from old Hippie
-	explanation_text = "Ensure there are no corpses on the station outside of the morgue when the shift ends."
+	explanation_text = "Ensure the Medbay has been cleaned of any corpses when the shift ends."
 	jobs = "chiefmedicalofficer,geneticist,medicaldoctor"
 
 /datum/objective/crew/morgue/check_completion()
-	for(var/mob/living/carbon/human/H in GLOB.mob_list)
-		if(H.stat == DEAD && (H.z in SSmapping.levels_by_trait(ZTRAIT_STATION)))
-			if(get_area(H) != /area/medical/morgue)
-				return FALSE
+	for(var/mob/living/carbon/human/H in GLOB.mob_living_list)
+		if(H.stat == DEAD && ((get_area(H) == typesof(/area/medical/medbay/central)) || (get_area(H) == typesof(/area/medical/patients_rooms))))
```

Comment:
Should just check /area/medical in my opinion

Reply:
Nevermind turns out chemistry and a bunch of other things are medical too, keep it how it is